tasks/utils.js

```diff
@@ -274,6 +274,20 @@ const logErrorMessage = err => {
   console.log(`${message || err}\n\n`);
 };
 
+const bustCache = file => {
+  const ext = path.extname(file);
+  const baseName = path.basename(file, ext);
+  const bustedName = `${baseName}.${Date.now()}${ext}`;
```

Comment:
@AndrewGibson27, how do you feel about using `Date.now()`? It's a pattern I've seen before, but I'm not sure if there are any concerns with this approach.